### ISSUE TITLE
Update runtime to 21.08

### ISSUE THREAD

**Issue by LeandroStanger** [flathub/net.pcsx2.PCSX2#30]

**Comment by Mershl** [comment]
Yes, please.

**Comment by rowbawts** [comment]
Bump, this runtime is EOL now.

**Comment by HugLifeTiZ** [comment]
I'm working on a PR that handles this as well as updating the dependencies and updating PCSX2 to latest git, but it looks like PCSX2 bundles a mirror of XZ for liblzma, and even though flatpak-builder is able to pull down the submodule, for some reason, CMake tries to do it again and fails:

```
Cloning into '/home/tiz/prj/Flathub/net.pcsx2.PCSX2/.flatpak-builder/build/pcsx2-7/3rdparty/xz/xz'...
fatal: remote transport reported error
fatal: clone of 'file:///home/tiz/prj/Flathub/net.pcsx2.PCSX2/.flatpak-builder/git/https_github.com_PCSX2_xz.git' into submodule path '/home/tiz/prj/Flathub/net.pcsx2.PCSX2/.flatpak-builder/build/pcsx2-7/3rdparty/xz/xz' failed
Failed to clone '3rdparty/xz/xz'. Retry scheduled
Cloning into '/home/tiz/prj/Flathub/net.pcsx2.PCSX2/.flatpak-builder/build/pcsx2-7/3rdparty/xz/xz'...
fatal: remote transport reported error
fatal: clone of 'file:///home/tiz/prj/Flathub/net.pcsx2.PCSX2/.flatpak-builder/git/https_github.com_PCSX2_xz.git' into submodule path '/home/tiz/prj/Flathub/net.pcsx2.PCSX2/.flatpak-builder/build/pcsx2-7/3rdparty/xz/xz' failed
Failed to clone '3rdparty/xz/xz' a second time, aborting
Error: module pcsx2: Child process exited with code 1
```

It seems like it should work since the submodule URL was rewritten to be a local location, but it doesn't say why the clone failed, just "remote transport reported error".

**Comment by j8r** [comment]
Solved by #33
